src/test/scala/dxWDL/compiler/NativeTest.scala

```diff
@@ -177,23 +185,57 @@ class NativeTest extends FlatSpec with Matchers with BeforeAndAfterAll {
   it should "Native compile a workflow with one level nesting" taggedAs (NativeTestXX, EdgeTest) in {
     val path = pathFromBasename("nested", "two_levels.wdl")
     val retval = Main.compile(
-        path.toString :: "--force" :: "--execTree" :: cFlags
+        path.toString :: "--force" :: "--execTree" :: "json" :: cFlags
     )
     retval shouldBe a[Main.SuccessfulTerminationTree]
 
     inside(retval) {
-      case Main.SuccessfulTerminationTree(treeJs, treePretty) =>
-        treeJs.asJsObject.getFields("name", "kind", "stages") match {
-          case Seq(JsString(name), JsString(kind), JsArray(stages)) =>
-            name shouldBe ("two_levels")
-            kind shouldBe ("workflow")
-            stages.size shouldBe (3)
-          //System.out.println(treePretty)
-          case other =>
-            throw new Exception(s"tree representation is wrong ${treeJs}")
+      case Main.SuccessfulTerminationTree(pretty) =>
+        pretty match {
+          case Left(str) => false // should not produce a pretty string
+          case Right(treeJs) => {
+            treeJs.asJsObject.getFields("name", "kind", "stages") match {
+              case Seq(JsString(name), JsString(kind), JsArray(stages)) =>
+                name shouldBe ("two_levels")
+                kind shouldBe ("workflow")
+                stages.size shouldBe (3)
+              case other =>
+                throw new Exception(s"tree representation is wrong ${treeJs}")
+            }
+          }
         }
     }
   }
+  
```

Comment:
It is preferable to ignore this test (`ignore should "Display ..."`), if you aren't happy with it.